src/lib/csv/pacbio.js

```diff
@@ -71,8 +74,23 @@ const eachRecord = (csv, callback) => {
     trim: true,
     info: true,
     cast,
-    onRecord: filterUndefinedValues,
-  }).forEach(callback)
+  }).map(filterUndefinedValues)
+
+  let retRecords = []
```

Comment:
As we discussed yesterday, the eachRecord function has been refactored to ensure that existing cases are not affected. Currently, eachRecord does not return any values, nor is it used in a validation mode where a failure to return a value would halt further processing.
Therefore, if eachRecord does not return a value (as is the case in existing implementations), the method will continue executing and simply return the parsed CSV records. 
Additionally, I have not converted it into a reduce function because the current implementation allows for an early return in case of invalid results.
I hope this makes sense.